src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -42,19 +42,19 @@
 
 // Dummy constructor
 ReservedSpace::ReservedSpace() : _base(nullptr), _size(0), _noaccess_prefix(0),
-    _alignment(0), _special(false), _fd_for_heap(-1), _executable(false) {
+    _alignment(0), _fd_for_heap(-1), _special(false), _executable(false), _nmt_flag(mtNone) {
```

Comment:
Isn't just `_flag` or `_memflag` sufficient as a name for `ReservedSpace`? We don' use `nmt_flag` anywhere else in the codebase.

Reply:
Yes, I would keep consistency with existing code, and maybe later rename all in one followup change

Reply:
Done.